app/1.2.x/admin-api.md

```diff
@@ -613,13 +635,38 @@ HTTP 200 OK
         "connections_reading": 0,
         "connections_writing": 1,
         "connections_waiting": 0
-    },
-    "database": {
-        "reachable": true
     }
 }
 ```
 
+* `memory`: Metrics about the memory usage.
+    * `workers_lua_vms`: An array with all workers of the Kong node, where each
+      entry contains:
+        * `http_allocated_gc`: HTTP submodule's Lua virtual machine's memory
+          usage information, as reported by `collectgarbage("count")`, for every
+          active worker, i.e. a worker that received a proxy call in the last 10
+          seconds.
+        * `pid`: worker's process identification number.
+    * `lua_shared_dicts`: An array of information about dictionaries that are
+      shared with all workers in a Kong node, where each array node contains how
+      much memory is dedicated for the specific shared dictionary (`capacity`)
+      and how much of said memory is in use (`allocated_slabs`).
+      These shared dictionaries have least recent used (LRU) eviction
+      capabilities, so a full dictionary, where `allocated_slabs == capacity`,
+      will work properly. However for some dictionaries, e.g. cache HIT/MISS
+      shared dictionaries, increasing their size can be beneficial for the
+      overall performance of a Kong node.
+    * The memory usage unit and precision can be changed using the querystring
+      arguments `unit` and `scale`:
+        * `unit`: one of `b/B`, `k/K`, `m/M`, `g/G`, which will return results
+          in bytes, kibibytes, mebibytes, or gibibytes, respectively. When
+          "bytes" are requested, the memory values in the response will have a
+          number type instead of string. Defaults to `k`.
```

Comment:
The default is `m` (my bad, for the comment I posted earlier was not taking subsequent changes I made to the endpoint in consideration).

Reply:
(As demonstrated by the above example response)

Reply:
Fixed!